plugwise/helper.py

```diff
@@ -237,11 +237,11 @@ async def _request_validate(self, resp, method) -> etree:
 
     async def _request(
         self,
-        command: str,
+        command,
         retry=3,
         method="get",
-        data: str | None = None,
-        headers: dict[str, str] | None = None,
+        data=None,
+        headers=None,
```

Comment:
This is going the opposite direction. **Always** type arguments of methods.

Reply:
OK! Then I need to get working again :)

Reply:
Ok done. How do the updates look?